### ISSUE TITLE
Unable to npm start project

### ISSUE THREAD

**Issue by andrewkoung** [Automattic/wp-calypso#31059]
Hi! I'm trying to learn about this project and hopefully pick up some issues, but I'm having trouble starting the project. I've followed the instructions in the README.md only to get this: 

#### Browser / OS version
Windows 10

#### Screenshot / Video
https://gyazo.com/20e9a297709508ab5dee78ba7e5f645c

**Comment by caprado** [comment]
The error is on the scripts replace everything in package.json with this. Let me know if it runs. I've added ./ in front of all scripts that start with "node_modules"


```json
{
  "name": "wp-calypso",
  "version": "0.17.0",
  "description": "A pure REST-API and JS based version of the WordPress.com admin",
  "private": true,
  "license": "GPL-2.0-or-later",
  "repository": {
    "type": "git",
    "url": "https://github.com/Automattic/wp-calypso.git"
  },
  "main": "index.js",
  "bin": {
    "calypso-sdk": "./bin/sdk-cli.js"
  },
  "browserslist": [
    "last 2 versions",
    "Safari >= 10",
    "iOS >= 10",
    "not ie <= 10",
    "> 1%"
  ],
  "dependencies": {
    "@automattic/tree-select": "file:./packages/tree-select",
    "@babel/cli": "7.2.3",
    "@babel/core": "7.2.2",
    "@babel/plugin-proposal-class-properties": "7.3.0",
    "@babel/plugin-proposal-export-default-from": "7.2.0",
    "@babel/plugin-proposal-export-namespace-from": "7.2.0",
    "@babel/plugin-syntax-dynamic-import": "7.2.0",
    "@babel/plugin-syntax-jsx": "7.2.0",
    "@babel/plugin-transform-runtime": "7.2.0",
    "@babel/polyfill": "7.2.5",
    "@babel/preset-env": "7.3.1",
    "@babel/preset-react": "7.0.0",
    "@babel/runtime": "7.3.1",
    "@wordpress/a11y": "2.0.2",
    "@wordpress/api-fetch": "2.2.8",
    "@wordpress/babel-plugin-import-jsx-pragma": "1.1.3",
    "@wordpress/blob": "2.1.0",
    "@wordpress/block-library": "2.2.16",
    "@wordpress/blocks": "6.0.7",
    "@wordpress/components": "7.0.8",
    "@wordpress/compose": "3.0.1",
    "@wordpress/core-data": "2.0.17",
    "@wordpress/data": "4.2.1",
    "@wordpress/date": "3.0.1",
    "@wordpress/deprecated": "2.0.5",
    "@wordpress/dom": "2.0.8",
    "@wordpress/edit-post": "3.1.11",
    "@wordpress/editor": "9.0.11",
    "@wordpress/element": "2.1.9",
    "@wordpress/format-library": "1.2.14",
    "@wordpress/hooks": "2.0.5",
    "@wordpress/i18n": "3.1.1",
    "@wordpress/keycodes": "2.0.6",
    "@wordpress/notices": "1.1.3",
    "@wordpress/nux": "3.0.9",
    "@wordpress/plugins": "2.0.11",
    "@wordpress/rich-text": "3.0.7",
    "@wordpress/token-list": "1.1.0",
    "@wordpress/url": "2.3.3",
    "@wordpress/viewport": "2.1.1",
    "acorn": "6.1.0",
    "autoprefixer": "9.4.4",
    "autosize": "4.0.2",
    "babel-loader": "8.0.5",
    "babel-plugin-add-module-exports": "1.0.0",
    "babel-plugin-transform-class-properties": "6.24.1",
    "babel-plugin-transform-export-extensions": "6.22.0",
    "body-parser": "1.18.3",
    "bounding-client-rect": "1.0.5",
    "browser-filesaver": "1.1.1",
    "chalk": "2.4.2",
    "chokidar": "2.1.0",
    "chrono-node": "1.3.5",
    "circular-dependency-plugin": "5.0.2",
    "classnames": "2.2.6",
    "click-outside": "2.0.2",
    "clipboard": "2.0.4",
    "color-studio": "github:automattic/color-studio#1.0.1",
    "component-closest": "1.0.1",
    "component-file-picker": "0.2.1",
    "cookie": "0.3.1",
    "cookie-parser": "1.4.4",
    "copy-webpack-plugin": "4.6.0",
    "core-js": "2.6.5",
    "cpf_cnpj": "0.2.0",
    "create-react-class": "15.6.3",
    "creditcards": "3.0.1",
    "cross-env": "5.2.0",
    "css-loader": "2.1.0",
    "d3-array": "2.0.3",
    "d3-axis": "1.0.12",
    "d3-scale": "2.1.2",
    "d3-selection": "1.3.2",
    "d3-shape": "1.2.2",
    "debug": "3.2.6",
    "deep-freeze": "0.0.1",
    "diff": "3.5.0",
    "doctrine": "3.0.0",
    "dom-helpers": "3.4.0",
    "dom-scroll-into-view": "1.2.1",
    "dompurify": "1.0.8",
    "draft-js": "0.10.5",
    "duplicate-package-checker-webpack-plugin": "3.0.0",
    "email-validator": "2.0.4",
    "emoji-text": "0.2.6",
    "escape-regexp": "0.0.1",
    "escape-string-regexp": "1.0.5",
    "events": "3.0.0",
    "exports-loader": "0.7.0",
    "express": "4.16.4",
    "express-useragent": "1.0.12",
    "file-loader": "3.0.1",
    "filesize": "3.6.1",
    "flag-icon-css": "3.2.1",
    "flux": "3.1.3",
    "fontfaceobserver": "2.1.0",
    "fuse.js": "3.3.0",
    "get-video-id": "3.1.0",
    "gfm-code-blocks": "1.0.0",
    "globby": "9.0.0",
    "gridicons": "3.1.1",
    "gzip-size": "5.0.0",
    "hash.js": "1.1.7",
    "he": "1.2.0",
    "html-loader": "0.5.5",
    "html-to-react": "1.3.4",
    "html-webpack-plugin": "3.2.0",
    "i18n-calypso": "file:./packages/i18n-calypso",
    "ignore-loader": "0.1.2",
    "immutability-helper": "3.0.0",
    "immutable": "3.8.2",
    "imports-loader": "0.8.0",
    "inherits": "2.0.3",
    "is-my-json-valid": "2.19.0",
    "jquery": "1.12.3",
    "json-stable-stringify": "1.0.1",
    "jsx-to-string": "1.4.0",
    "keymaster": "1.6.2",
    "lerna": "3.13.0",
    "loader-utils": "1.2.3",
    "localforage": "1.7.3",
    "lodash": "4.17.11",
    "lru": "3.1.0",
    "lunr": "2.3.5",
    "mapbox-gl": "0.52.0",
    "markdown-it": "8.4.2",
    "marked": "0.5.2",
    "mini-css-extract-plugin-with-rtl": "github:Automattic/mini-css-extract-plugin-with-rtl",
    "mkdirp": "0.5.1",
    "moment": "2.24.0",
    "moment-timezone": "0.5.23",
    "morgan": "1.9.1",
    "node-sass": "4.11.0",
    "node-sass-package-importer": "5.3.0",
    "npm-run-all": "4.1.5",
    "objectpath": "1.2.2",
    "page": "1.11.4",
    "path-browserify": "1.0.0",
    "percentage-regex": "3.0.0",
    "phone": "git+https://github.com/Automattic/node-phone.git#v2.4.0-pre-1",
    "photon": "file:./packages/photon",
    "postcss-cli": "6.0.1",
    "postcss-custom-properties": "8.0.9",
    "postcss-loader": "3.0.0",
    "prismjs": "1.15.0",
    "prop-types": "15.7.2",
    "qrcode.react": "0.9.3",
    "qs": "6.6.0",
    "react": "16.8.3",
    "react-click-outside": "3.0.1",
    "react-day-picker": "7.2.4",
    "react-dom": "16.8.3",
    "react-lazily-render": "1.1.0",
    "react-live": "1.12.0",
    "react-modal": "3.8.1",
    "react-pure-render": "1.0.2",
    "react-redux": "5.1.1",
    "react-transition-group": "2.5.0",
    "react-virtualized": "9.21.0",
    "redux": "4.0.1",
    "redux-form": "7.4.2",
    "redux-thunk": "2.3.0",
    "refx": "3.1.1",
    "rememo": "3.0.0",
    "resize-observer-polyfill": "1.5.1",
    "rtlcss": "2.4.0",
    "sass-loader": "7.1.0",
    "social-logos": "2.0.0",
    "socket.io-client": "2.2.0",
    "source-map": "0.7.3",
    "source-map-support": "0.5.10",
    "store": "2.0.12",
    "striptags": "2.2.1",
    "superagent": "3.8.3",
    "swiper": "4.4.6",
    "terser-webpack-plugin": "1.2.2",
    "textarea-caret": "3.1.0",
    "thread-loader": "2.1.2",
    "tinymce": "4.8.5",
    "to-title-case": "1.0.0",
    "tracekit": "0.4.5",
    "twemoji": "11.2.0",
    "url": "0.11.0",
    "uuid": "3.3.2",
    "valid-url": "1.0.9",
    "webpack": "4.29.5",
    "webpack-bundle-analyzer": "3.0.4",
    "webpack-cli": "3.2.3",
    "webpack-dev-middleware": "3.5.2",
    "webpack-rtl-plugin": "1.8.0",
    "wpcom": "5.4.2",
    "wpcom-oauth": "0.3.4",
    "wpcom-proxy-request": "5.0.2",
    "wpcom-xhr-request": "1.1.2",
    "yargs": "12.0.5"
  },
  "engines": {
    "node": "^10.13.0",
    "npm": "^6.1.0"
  },
  "scripts": {
    "preanalyze-bundles": "cross-env-shell CALYPSO_ENV=production EMIT_STATS=true NODE_ARGS=--max_old_space_size=8192 npm run -s build-client",
    "analyze-bundles": "webpack-bundle-analyzer stats.json public -h 127.0.0.1 -p 9898 -s gzip",
    "analyze-css": "node bin/analyze-css.js",
    "autoprefixer": "postcss -u autoprefixer -r --no-map",
    "postcss": "postcss -r",
    "prebuild": "npm run -s install-if-deps-outdated",
    "build": "npm run build-css && npm run build-packages && run-p -s build-devdocs:* build-server build-client-if-prod build-client-if-desktop",
    "prebuild-css": "node bin/build-custom-properties-css.js",
    "build-css": "run-p -s build-css:*",
    "build-css:debug": "node-sass --importer ./node_modules/node-sass-package-importer/dist/cli.js --include-path client --source-map \"public/style-debug.css.map\" assets/stylesheets/style.scss public/style-debug.css && npm run -s postcss -- public/style-debug.css && rtlcss public/style-debug.css public/style-debug-rtl.css",
    "build-css:directly": "node-sass --importer ./node_modules/node-sass-package-importer/dist/cli.js --include-path client assets/stylesheets/directly.scss public/directly.css --output-style compressed && npm run -s postcss -- public/directly.css",
    "build-css:editor": "node-sass --importer ./node_modules/node-sass-package-importer/dist/cli.js --include-path client assets/stylesheets/editor.scss public/editor.css --output-style compressed && npm run -s postcss -- public/editor.css",
    "build-css:style": "node-sass --importer ./node_modules/node-sass-package-importer/dist/cli.js --include-path client assets/stylesheets/style.scss public/style.css --output-style compressed && npm run -s postcss -- public/style.css && rtlcss public/style.css public/style-rtl.css",
    "build-devdocs:components-usage-stats": "cross-env-shell NODE_PATH=$NODE_PATH:server:client:. npm run -s build-devdocs:components-usage-stats:_env",
    "build-devdocs:components-usage-stats:_env": "node server/devdocs/bin/generate-components-usage-stats.js \"client/**/*.js\" \"client/**/*.jsx\" \"!**/docs/**\" \"!**/test/**\" \"!**/docs-example/**\"",
    "build-devdocs:index": "cross-env-shell NODE_PATH=$NODE_PATH:server:client:. npm run -s build-devdocs:index:_env",
    "build-devdocs:index:_env": "node server/devdocs/bin/generate-devdocs-index",
    "build-docker": "node bin/build-docker.js",
    "prebuild-server": "mkdirp build",
    "build-server": "cross-env-shell CALYPSO_SERVER=true NODE_PATH=$NODE_PATH:server:client:. webpack --display errors-only --config webpack.config.node.js",
    "build-client": "cross-env-shell CALYPSO_CLIENT=true NODE_PATH=$NODE_PATH:server:client:. node $NODE_ARGS ./node_modules/webpack/bin/webpack.js --display errors-only",
    "build-client-if-prod": "node -e \"process.env.CALYPSO_ENV === 'production' && process.exit(1)\" || cross-env-shell NODE_ARGS=--max_old_space_size=8192 npm run build-client",
    "build-client-if-desktop": "node -e \"process.env.CALYPSO_ENV === 'desktop' && process.exit(1)\" || npm run build-client",
    "build-packages": "npx lerna run build --stream",
    "sdk": "node bin/sdk-cli.js",
    "clean": "npm run -s clean:build && npm run -s clean:devdocs && npm run -s clean:public && npm run -s clean:packages",
    "clean:build": "npx rimraf build server/bundler/*.json .babel-cache",
    "clean:devdocs": "npx rimraf server/devdocs/search-index.js server/devdocs/proptypes-index.json server/devdocs/components-usage-stats.json",
    "clean:packages": "npx rimraf packages/*/dist",
    "clean:public": "npx rimraf public/*.{css,css.map,js,js.map} public/images/*.{gif,jpg,png,svg} public/{sections,sections-rtl,vendors~*}",
    "distclean": "npm run -s clean && npx rimraf ./node_modules packages/*/node_modules",
    "docker": "docker run -it --name wp-calypso --rm -p 80:3000 wp-calypso",
    "eslint-branch": "node bin/eslint-branch.js",
    "install-if-deps-outdated": "node bin/install-if-deps-outdated.js",
    "install-if-no-packages": "node bin/install-if-no-packages.js",
    "postinstall": "npm run build-packages",
    "lint": "run-s -s lint:*",
    "lint:config-defaults": "node server/config/validate-config-keys.js",
    "lint:css": "stylelint \"client/**/*.scss\" --syntax scss",
    "lint:js": "npm run -s install-if-deps-outdated && eslint .",
    "lint:mixedindent": "mixedindentlint --ignore-comments \"client/**/*.scss\" \"assets/**/*.scss\" \"**/*.js\" \"**/*.jsx\" \"!build/**\" \"!node_modules/**\" \"!public/**\" \"!client/config/index.js\"",
    "prestart": "npx check-node-version --package && npm run -s install-if-deps-outdated && node bin/welcome.js",
    "start": "npm run -s build",
    "poststart": "cross-env-shell CALYPSO_CLIENT=true NODE_PATH=$NODE_PATH:server:client:. node $NODE_ARGS --max_old_space_size=4096 build/bundle.js",
    "reformat-files": "./node_modules/.bin/prettier --ignore-path .eslintignore --write \"**/*.{js,jsx}\"",
    "pretest": "npm run -s install-if-deps-outdated",
    "test": "run-s -s test-client test-packages test-server",
    "pretest-client": "npm run -s pretest",
    "test-client": "jest -c=test/client/jest.config.js",
    "test-client:watch": "npm run -s test-client -- --watch",
    "pretest-integration": "npm run -s pretest",
    "test-integration": "jest -c=test/integration/jest.config.js",
    "test-integration:watch": "npm run -s test-integration -- --watch",
    "test-packages": "jest -c=test/packages/jest.config.js",
    "test-packages:watch": "jest -c=test/packages/jest.config.js --watch",
    "pretest-server": "npm run -s pretest",
    "test-server": "jest -c=test/server/jest.config.js",
    "test-server:coverage": "npm run -s test-server -- --coverage",
    "test-server:watch": "npm run -s test-server -- --watch",
    "translate": "i18n-calypso --format pot --output-file ./calypso-strings.pot -k translate,__,_x,_n,_nx -e date '**/*.js' '**/*.jsx' '!build/**' '!public/**' '!**/test/**' '!node_modules/**' '!**/node_modules/**'",
    "update-deps": "npx rimraf npm-shrinkwrap.json && npm run -s distclean && npm i && npm shrinkwrap && replace --silent 'http://' 'https://' . --recursive --include='npm-shrinkwrap.json,package-lock.json'",
    "postshrinkwrap": "node -e \"fs.utimesSync( './node_modules', new Date(), new Date() );\"",
    "prewhybundled": "cross-env-shell CALYPSO_ENV=production EMIT_STATS=withreasons NODE_ARGS=--max_old_space_size=8192 npm run -s build-client",
    "whybundled": "whybundled stats.json"
  },
  "devDependencies": {
    "@automattic/babel-plugin-i18n-calypso": "file:./packages/babel-plugin-i18n-calypso",
    "@babel/plugin-transform-react-jsx": "7.3.0",
    "@wordpress/babel-plugin-makepot": "2.1.3",
    "babel-eslint": "10.0.1",
    "babel-jest": "24.1.0",
    "babel-plugin-dynamic-import-node": "2.2.0",
    "chai": "4.2.0",
    "chai-enzyme": "1.0.0-beta.1",
    "check-node-version": "3.2.0",
    "enzyme": "3.9.0",
    "enzyme-adapter-react-16": "1.9.1",
    "enzyme-to-json": "3.3.5",
    "eslint": "5.14.1",
    "eslint-config-prettier": "4.0.0",
    "eslint-config-wpcalypso": "file:./packages/eslint-config-wpcalypso",
    "eslint-plugin-import": "2.14.0",
    "eslint-plugin-jest": "22.3.0",
    "eslint-plugin-jsx-a11y": "6.2.1",
    "eslint-plugin-react": "7.12.4",
    "eslint-plugin-wpcalypso": "file:./packages/eslint-plugin-wpcalypso",
    "glob": "7.1.3",
    "husky": "1.3.1",
    "jest": "24.1.0",
    "jest-docblock": "24.0.0",
    "jest-junit": "6.3.0",
    "md5-file": "4.0.0",
    "mixedindentlint": "1.2.0",
    "mockdate": "2.0.2",
    "moment-timezone-data-webpack-plugin": "1.0.3",
    "nock": "10.0.6",
    "npm-merge-driver": "2.3.5",
    "prettier": "https://github.com/Automattic/wp-prettier/releases/download/wp-1.16.4/wp-prettier-1.16.4.tgz",
    "react-test-renderer": "16.8.3",
    "readline-sync": "1.4.9",
    "replace": "1.0.1",
    "rimraf": "2.6.3",
    "sinon": "7.1.1",
    "sinon-chai": "3.3.0",
    "socket.io": "2.2.0",
    "stacktrace-gps": "3.0.2",
    "stylelint": "9.10.1",
    "supertest": "3.4.2",
    "webpack-filter-warnings-plugin": "1.2.1",
    "webpack-hot-middleware": "2.24.3",
    "whybundled": "1.4.2"
  },
  "optionalDependencies": {
    "fsevents": "2.0.1"
  },
  "husky": {
    "hooks": {
      "pre-commit": "npm run -s install-if-no-packages && node bin/pre-commit-hook.js",
      "pre-push": "npm run -s install-if-no-packages && node bin/pre-push-hook.js"
    }
  }
}
```

**Comment by andrewkoung** [comment]
@caprado I'm getting this now. 
https://gyazo.com/fd10ce5b5aed9a44510da476856f94fa

**Comment by caprado** [comment]
That's the package `@automattic/tree-select`. 

Try to run `npm install` so it downloads any missing node_modules.

**Comment by andrewkoung** [comment]
I've tried to npm install, but it's seem showing the same message.

**Comment by kktam** [comment]
I get a very similar error message on Windows 10 professional.

A portion of the error message is :

```
Child mini-css-extract-plugin node_modules/css-loader/dist/cjs.js??ref--6-1!node_modules/postcss-loader/src/index.js??ref--6-2!node_modules/sass-loader/lib/loader.js??ref--6-3!packages/calypso-ui/dist/esm/screen-reader-text/style.scss:

    ERROR in ./packages/calypso-ui/dist/esm/screen-reader-text/style.scss (./node_modules/css-loader/dist/cjs.js??ref--6-1!./node_modules/postcss-loader/src??ref--6-2!./node_modules/sass-loader/lib/loader.js??ref--6-3!./packages/calypso-ui/dist/esm/screen-reader-text/style.scss)
    Module build failed (from ./node_modules/sass-loader/lib/loader.js):

    // Text meant only for screen readers
    ^
          File to import not found or unreadable: C:Usersnodewordpresswp-calypso
    ssetsstylesheetsshared_utils.scss.
          in C:\Users\n\code\wordpress\wp-calypso\packages\calypso-ui\dist\esm\screen-reader-text\style.scss (line 1, column 1)
    Error:
    // Text meant only for screen readers
    ^
          File to import not found or unreadable: C:Usersnodewordpresswp-calypso
    ssetsstylesheetsshared_utils.scss.
          in C:\Users\n\code\wordpress\wp-calypso\packages\calypso-ui\dist\esm\screen-reader-text\style.scss (line 1, column 1)
        at options.error (C:\Users\n\code\wordpress\wp-calypso\node_modules\node-sass\lib\index.js:291:26)
i ｢wdm｣: Failed to compile.
```

This happens around webpack stage.

**Comment by lancewillett** [comment]
Comment for future readers or commenters: if you have experience building Calypso in a Windows server environment, please comment here and help each other troubleshoot.

**Comment by mattcousineau** [comment]
I am just going to leave this here for anyone who follows after me.  I was having a lot of strange build issues while trying to setup a **wp-calypso** workspace in Windows 10 - from missing dependencies that I knew were present, to simple missing CSS warnings.   

I found that the simple overstep of forgetting to run GIT Bash as an administrator was causing me a lot of headache.   After running as admin, many of these issues were resolved.  I hope this helps someone!
